### DISCUSSION TITLE
When I upgraded the target to 1.8.0.220, the program stopped working

### DISCUSSION THREAD

**Discussion by babyblue2** [nanoframework/Home#1051] [General]
Target: ESP32 REV3

> Link failure: some assembly references cannot be resolved!!
> Assembly: GpioRest (1.0.0.0) needs assembly 'nanoFramework.WebServer' (1.0.0.3)
> **Assembly: nanoFramework.WebServer (1.0.0.3) needs assembly 'System.Net' (1.8.1.3)**
> Error: a3000000

**System.Net1.8.  x does not work on Target 1.8.0.220, but Nanoframework.  webServer depends on it.**

What should I do?

**Comment by alberk8** [comment]
@babyblue2 ,  use the nanoframework.WebServer nuget from **Stable**.   Don't use the preview version.

**Reply by babyblue2** [reply]
I'm already using the stable version, Nanoframework.webServer ver 1.0.0
